sources/packages/backend/libs/sims-db/src/entities/institution.model.ts

```diff
@@ -63,11 +63,20 @@ export class Institution extends RecordDataModel {
     name: "website",
   })
   website: string;
+
   @Column({
     name: "regulating_body",
   })
   regulatingBody: string;
 
+  /**
+   * Other regulating body
```

Comment:
Period missing in the comment. Also please try to use the same comment used for the column.

![image](https://user-images.githubusercontent.com/54600590/222004569-3bb113b8-839d-465d-837d-b5f65fa63c31.png)